src/app/portlets/dot-edit-page/layout/layout-properties-field/dot-layout-properties-item/dot-layout-properties-item.component.ts

```diff
@@ -0,0 +1,71 @@
+import { DotLayoutSidebarComponent } from './../dot-layout-property-sidebar/dot-layout-property-sidebar.component';
+import { ControlValueAccessor, NG_VALUE_ACCESSOR } from '@angular/forms';
+import { Component, Input, Output, EventEmitter, forwardRef, HostListener, ViewChild } from '@angular/core';
+
+@Component({
+    selector: 'dot-layout-properties-item',
+    templateUrl: './dot-layout-properties-item.component.html',
+    styleUrls: ['./dot-layout-properties-item.component.scss'],
+    providers: [
+        {
+            multi: true,
+            provide: NG_VALUE_ACCESSOR,
+            useExisting: forwardRef(() => DotLayoutPropertiesItemComponent)
+        }
+    ]
+})
+export class DotLayoutPropertiesItemComponent implements ControlValueAccessor {
+    @Input() label: string;
+    @Output() change: EventEmitter<boolean> = new EventEmitter();
+
+    value: boolean;
+
+    constructor() {}
+
+    propagateChange = (_: any) => {};
+
+    @HostListener('click', ['$event'])
+    onClick() {
+        this.value = !this.value;
+        this.propagateChange(this.value);
+        this.change.emit(this.value);
+    }
+
+    /**
+     * Write a new value to the property item
+     * @param {boolean} value
+     * @memberof DotLayoutPropertiesItemComponent
+     */
+    writeValue(value: boolean): void {
+        if (value) {
+            this.value = value;
+        }
+    }
+
+    /**
+     * Check item and set value to true
+     * @memberof DotLayoutPropertiesItemComponent
+     */
+    setChecked() {
```

Comment:
not sure what are you doing here, but won't be easer just to toggle it such as

toggle () { this.value = !this.value; }